speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp

```diff
@@ -1530,6 +1532,9 @@ void cGrBoard::refreshBoard(tSituation *s, const cGrFrameInfo* frameInfo,
       grDispSplitScreenIndicator();
   }
 
+  // For now: display an intervention every frame.
+  grDispIntervention();
```

Comment:
Same here: // SIMULATED DRIVING ASSISTANCE CHANGE: For now: display an intervention every frame.